samples/helloworld/pom.xml

```diff
@@ -36,7 +36,12 @@
     <dependency>
       <groupId>org.apache.logging.log4j</groupId>
       <artifactId>log4j-slf4j-impl</artifactId>
-      <version>2.15.0</version>
+      <version>2.17.0</version>
+    </dependency>
+    <dependency>
```

Comment:
same here could you remove one of the org.apache.logging.log4j  dependency declaration?
Thx

Reply:
removed